assets/src/js/components/SiteSwitcher.js

```diff
@@ -58,7 +58,7 @@ class SiteSwitcher extends Component {
 
     // otherwise, render list of sites + add button
     return (
-        <li class={`sites ${state.isExpanded ? 'expanded' : ''}`} onClick={this.toggleExpanded} onMouseEnter={this.expand} onMouseLeave={this.collapse}>
+        <li style={{display: props.visble ? '' : 'none'}} class={`sites ${state.isExpanded ? 'expanded' : ''}`} onClick={this.toggleExpanded} onMouseEnter={this.expand} onMouseLeave={this.collapse}>
```

Comment:
Can you change this property name to `visible`?

Reply:
Fixed in a1b0321.